java/code/src/com/redhat/rhn/domain/contentmgmt/ContentFilterEntity.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2025 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ */
+
+/**
+ * Marker interface for entities compatible with {@link ContentFilter}.
+ * Implemented by {@link Errata}, {@link Package}, {@link Module}, etc.,
+ * to enforce type-safe filtering operations.
+ */
+
+package com.redhat.rhn.domain.contentmgmt;
+
+public interface ContentFilterEntity {
+}
```

Comment:
The javadoc is misplaced, not sure if it will be picked up by the javadoc tool/IDE if placed like it is now. It should be:

```suggestion
package com.redhat.rhn.domain.contentmgmt;

/**
 * Marker interface for entities compatible with {@link ContentFilter}.
 * Implemented by {@link Errata}, {@link Package}, {@link Module}, etc.,
 * to enforce type-safe filtering operations.
 */
public interface ContentFilterEntity {
}
```